python/housinginsights/sources/census.py

```diff
@@ -1,61 +1,123 @@
+                        "acs5_2009_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2010_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2011_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2012_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2013_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2014_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M",
+                        "acs5_2015_moe" : "NAME,B01003_001M,B25057_001M,B25058_001M,B25059_001M"
+                       }	
+
+    def get_data(self, unique_data_ids=None, sample=False, output_type = 'csv', **kwargs):
+        '''
+        Gets all data sources associated with this ApiConn class. 
+
+        sample does not apply to this apiconn object because api doesn't let us request only some data. 
+        '''
+        if unique_data_ids == None:
+            unique_data_ids = self._available_unique_data_ids
+
+        for u in unique_data_ids:
+            if (u not in self._available_unique_data_ids):
+                #TODO Change this error type to a more specific one that should be handled by calling function inproduction
+                #We will want the calling function to continue with other data sources instead of erroring out. 
+                logging.info("  The unique_data_id '{}' is not supported by the CensusApiConn".format(u))
+            else:
+                result = self.get(self._urls[u], params={'key':self.census_api_key, 'get':self._fields[u], 'for': 'tract:*', 'in': 'state:11'})
+                
+                if result.status_code != 200:
+                    err = "An error occurred during request: status {0}"
+                    #TODO change this error type to be handleable by caller
+                    raise Exception(err.format(result.status_code))
+                
+                content = result.text
+
+                if output_type == 'stdout':
+                    print(content)
+
+                elif output_type == 'csv':
+                    jsondata=json.loads(content)
+                    self.result_to_csv(jsondata[0], jsondata[1:],  self.output_paths[u])
+                
+                #Can't yield content if we get multiple sources at once
+                if len(unique_data_ids) == 1:
+                    return content
+
+
+
+if __name__ == '__main__':
+    
+    # Pushes everything from the logger to the command line output as well.
+    log_path = os.path.abspath(os.path.join(os.path.dirname(__file__),os.pardir,os.pardir,"logs","sources.log"))
+    logging.basicConfig(filename=log_path, level=logging.DEBUG)
+    logging.getLogger().addHandler(logging.StreamHandler())
+
+    api_conn = CensusApiConn()
+    unique_data_ids = None #['crime_2013']
+    api_conn.get_data(unique_data_ids)
```

Comment:
We want to also be able to run this from the /scripts/get_api_data.py file. The only small change required there is to add it to the list of available modules. I'll make this change after merging too - just an FYI for next time.